### PR TITLE
[traffic-gen-in-vm] E2E: Fix output fields assertions

### PR DESCRIPTION
**Pull request by orelmisan**
Cherry-pick PR #120.

### CODE REVIEW

**Comment by orelmisan** [comment]
Lets wait for PR #118 to be merged before merging this PR.

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/2adcbfda4950c2eec34ef245e6129ed632f7fb4f..c436e6f6810a187033afb1c7e038497cda92be69): Rabase

**Review by RamLavi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
